cluster-autoscaler/core/scale_up.go

```diff
@@ -417,6 +477,27 @@ func applyMaxClusterCoresMemoryLimits(newNodes int, coresTotal, memoryTotal, max
 				"max memory already reached")
 		}
 	}
+
+	gpuType, gpuCount, err := gpu.GetNodeTargetGpus(nodeInfo.Node(), nodeGroup)
+	if err != nil {
+		// This is not very elegant, but it allows us to proceed even if we're
+		// unable to compute cpu/memory limits (not breaking current functionality)
```

Comment:
cpu/memory -> gpu

Also, isn't this the same case as with calculating total cluster GPUs (see "not safe to just let NAP go on GPU rampage" comment)? @MaciekPytel